app/controllers/application_controller.rb

```diff
@@ -27,6 +28,22 @@ def render(*args)
 
   private
 
+  # filter out restricted products, defined in flow coonsole
+  # https://console.flow.io/:organization/restrictions
+  def flow_filter_products
```

Comment:
this approach would require keeping all of the restrictions in sync, whereas what we are hoping will happen is:

1. Store the `status` field as part of
https://docs.flow.io/module/localization/resource/experiences#get-organization-experiences-items

2. when selecting items, add a "status='included'" filter